spellmonitor.lic

```diff
@@ -127,7 +127,7 @@ spell_action = proc do |server_string|
       # Spell fading away
       spell = Regexp.last_match[:spell]
       duration = 0
-    when /(?<spell>[^<>]+?)\s+\(indefinite|OM\)/i
+    when /(?<spell>[^<>]+?)\s+\((?<duration>indefinite|om)\)/i
```

Comment:
I mistakenly read `\(` as the start of a regular `(` but, in fact, that was an escaped literal parenthesis, not the start of regex group. This fix puts `indefinite|om` inside a regex group so the `|` is treated as logical OR.